File: kalkulator.py
class Kalkulator:
    def perjumlahan(self, a, b):
        return a + b

    def pengurangan(self, a, b):
        return a - b

    def perkalian(self, a, b):
        return a * b

    def pembagian(self, a, b):
        if b == 0:
            return "Tidak bisa pembagian dengan nol"
        else:
            return a / b

Kalkulator = Kalkulator()

while True:
    print("Pilih Operasi:")
    print("1. Perjumlahan")
    print("2. Pengurangan")
    print("3. Perkalian")
    print("4. Pembagian")
    print("5. Keluar")

    pilih = input("Masukkan Pilihan (1/2/3/4/5): ")

    if pilih in ('1', '2', '3', '4'):
        num1 = float(input("Masukkan bilangan pertama: "))
        num2 = float(input("Masukkan bilangan kedua: "))

        if pilih == '1':
            print(num1, "+", num2, "=", Kalkulator.perjumlahan(num1, num2))

        elif pilih == '2':
            print(num1, "-", num2, "=", Kalkulator.pengurangan(num1, num2))

        elif pilih == '3':
            print(num1, "*", num2, "=", Kalkulator.perkalian(num1, num2))

        elif pilih == '4':
            print(num1, "/", num2, "=", Kalkulator.pembagian(num1, num2))

    elif pilih == '5':
        print("Program akan di close...")
        break

    else:
        print("Input Salah")

File: Kalkulator.java
import java.util.Scanner;

public class Kalkulator {

    public static void main(String[] args) {
         Scanner scanner = new Scanner(System.in);
          System.out.print("Masukkan Bilangan Pertama: ");
       double nilai1 = scanner.nextDouble();
       
          System.out.print("Masukkan Bilangan Kedua: ");
       double nilai2 = scanner.nextDouble();
       
        System.out.println("");
        System.out.println(" _______________");
        System.out.println("|Pilih operasi: |");
        System.out.println("|1. Penjumlahan |");
        System.out.println("|2. Pengurangan |");
        System.out.println("|3. Perkalian   |");
        System.out.println("|4. Pembagian   |");
        System.out.println("|_______________|");
        System.out.println("");
        System.out.print("Masukkan Pilihan(1/2/3/4): ");
        
        int pilihan = scanner.nextInt();
        
   
       

        
           double hasil = 0;
       
       switch (pilihan) {
           case 1:
               hasil = nilai1 + nilai2;
           break;
           
           case 2:
               hasil = nilai1 - nilai2;
           break;
           case 3:
               hasil = nilai1 * nilai2;
           break;
           case 4:
               hasil = nilai1 / nilai2;
           break;
           
           default:
               System.out.println("Pilihan Salah");
           
       }
       
        System.out.println("");
        System.out.println("Hasil: "+nilai1+" + "+nilai2+" = "+hasil);
    }
    
}
